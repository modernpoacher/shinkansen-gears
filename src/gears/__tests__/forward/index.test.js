import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

import Forward from 'shinkansen-gears/gears/forward'

Enzyme.configure({ adapter: new Adapter() })

describe('shinkansen-gears/gears/forward', () => {
  it('renders', () => {
    expect(shallow(<Forward pathname={'MOCK PATHNAME'} />))
      .toMatchSnapshot()
  })
})
