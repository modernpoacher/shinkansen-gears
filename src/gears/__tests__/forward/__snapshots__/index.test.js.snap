// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shinkansen-gears/gears/forward renders 1`] = `ShallowWrapper {}`;
